src/main/java/uk/gov/pay/products/validations/ProductRequestValidator.java

```diff
@@ -27,8 +27,13 @@ public ProductRequestValidator(RequestValidations requestValidations) {
                 FIELD_GATEWAY_ACCOUNT_ID,
                 FIELD_PAY_API_TOKEN,
                 FIELD_NAME,
-                FIELD_PRICE,
-                RETURN_URL);
+                FIELD_PRICE);
+
+        if(!errors.isPresent()){
+            if(!requestValidations.checkIfExists(payload, FIELD_RETURN_URL).isPresent()) {
+                errors = requestValidations.checkIsUrl(payload, FIELD_RETURN_URL);
+            }
```

Comment:
Don't we have to check if the `return_url` is `https` too? or are we doing it somewhere else?

Reply:
Validation is on front-end but can add the extra check